src/css/tabs/motors.css

```diff
@@ -62,10 +60,11 @@
     margin-left: 10px;
 }
 
-.tab-motors select .escprotocol {
+.tab-motors #mixer.mixerList,
+.tab-motors #escprotocol {
     width: 150px;
-    height: 150px;
```

Comment:
I see that the esc protocols have some unnecessary extra whitespace, it is unsightly for a selector that adapts to the length of the names:

![image](https://user-images.githubusercontent.com/43983086/158052015-55e3f04e-9e95-43be-8483-27d97a8fa73f.png)

Reply:
If the purpose is an alignment with another thing, i cant see this aligment nowhere.

Reply:
@limonspb likes to have the `mixerlist` as wide as the model image. `escprotocol` select is aligned with that. That means the mixerlist inhibits the same unnecessary extra whitespace.

Reply:
I can see mixerlist aligned with modle image, its is good. But escprotocol is not aligned with anithing.

Reply:
escprotocol is aligned with mixerlist

Reply:
How align with mixer box if it is another diferent and separate item?

Reply:
I think Asizon is correct. At least I wanted to alight only the mixer list with the image.

Reply:
In that case we should also use `fit-content` on `mixerlist` to remove unnecessary extra whitespace.
Converting the list to uppercase will make the selector align with the `mixer preview img`

![image](https://user-images.githubusercontent.com/8344830/158075852-6fbbf2ef-495d-41a6-b851-a74bff297240.png)

Reply:
In that case we should also use `fit-content` on `mixerlist` to remove unnecessary extra whitespace.
Converting the list to uppercase will make the selector align with the `mixer preview img`

![image](https://user-images.githubusercontent.com/8344830/158076130-10af210f-fa37-4d53-97f4-0c979d24d304.png)

Reply:
i am not sure which is better actually. SO it is up to you @haslinghuis